index.html

```diff
@@ -175,7 +175,11 @@ <h2 class="cardTitle">Skills</h2>
         </section>
       </div>
       <section class="contact-section">
-        <form method="post" action="https://formspree.io/f/mayknqrv" class="contact-container">
+        <form
+          method="post"
+          action="https://formspree.io/f/mayknqrv"
+          class="contact-container"
+        >
           <h3 class="sectionTitle">
             I'm always interested in hearing about new projects, so if you'd
             like to chat please get in touch.
```

Comment:
- [ ] Kindly try to stick to the Figma design as much as possible to make your project looks good, so try leveraging your background positioning, size, colors, and position of social media icons so that it matches with the Figma design. you got this 👍 

Figma design:
![image](https://user-images.githubusercontent.com/90524466/186462909-00eada3b-6b01-4cb3-85cf-704b1d5023ee.png)

your design:
![image](https://user-images.githubusercontent.com/90524466/186463085-714e013c-85bb-4fde-86cf-0f8237d72efc.png)